bundle/manifests/logging.openshift.io_clusterloggings.yaml

```diff
@@ -502,7 +502,9 @@ spec:
                         nullable: true
                         properties:
                           diskThresholdPercent:
-                            description: The threshold percentage of ES disk usage that when reached, old indices should be deleted (e.g. 75)
+                            description: The threshold percentage of ES disk usage
```

Comment:
@Red-GV  Looks like needless changes, probably related to your OS. (I have same with my MacOS)